corehq/form_processor/backends/sql/processor.py

```diff
@@ -33,6 +33,7 @@ def store_attachments(cls, xform, attachments):
                 attachment_id=uuid.uuid4(),
                 content_type=attachment.content_type,
             )
+            xform_attachment.blob_bucket = xform_attachment.blobdb_bucket()
```

Comment:
What do you think of moving this into `XFormAttachmentSQL.__init__`? Alternately, might make sense to move it into `AbstractAttachment.write_content`